cvat-data/webpack.config.js

```diff
@@ -0,0 +1,84 @@
+/* global
+    require:true,
+    __dirname:true,
+*/
+
+const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
+
+const moduleConf = {
+    rules: [
+        {
+            test: /.js?$/,
+            exclude: /node_modules/,
+            use: {
+                loader: 'babel-loader',
+                options: {
+                    presets: [
+                        ['@babel/preset-env', {
+                            targets: {
+                                chrome: 58,
```

Comment:
Let's set: ``targets: '> 2.5%', ``

It covers some versions of Chrome, Mozilla and Safari
And doesn't use polyfills from core-js for build, so, it should be faster. 
And we can remove core-js from ``package.json`` and ``package-lock.json``